scripts/jenkins/ardana/gerrit/build_test_package.py

```diff
@@ -338,7 +338,11 @@ def add_test_package(self, package):
                 service_file.truncate()
             # Run the osc service and commit the changes to OBS
             sh.osc('rm', glob.glob('%s*.obscpio' % package.name))
-            sh.osc('service', 'disabledrun')
+            env = os.environ.copy()
+            # make obs_scm work with a local path.
+            # otherwise it only works with remote URLs.
```

Comment:
Then please report upstream that we need it as a feature - we have the people inhouse who can fix that.

This section should then be marked as 'WORKAROUND' and link to the bug report (comment in code), so that we can later clean it up.

Btw. The upstream commit https://github.com/openSUSE/obs-service-tar_scm/pull/251/files#diff-d71a5775c1f5ed1522aa5a3b4dafec6dR174  says that they want to drop this workaround in favor of a stub. So we should make sure, that it is not dropped before having it as a feature.